### PR TITLE
Prepare to publish v6.1.2

### PR DESCRIPTION
**Pull request by daileytj**
#### Changes proposed in this Pull Request:
- Prepare to publish v6.1.2

### CODE REVIEW

**Review by EvanMcLaughlin-eaton — APPROVED** [review]

**Comment by daileytj** [comment]
@EvanMcLaughlin-eaton @emclaug2 I'm going to just merge this anyhow, butI need your approval off from the account with the codeowner role for the rest of these today

**Pull request merged**
